sdk/cosmos/azure-cosmos/test/test_query.py

```diff
@@ -501,6 +502,7 @@ def test_unsupported_queries(self):
             query_iterable = created_collection.query_items(query=query, enable_cross_partition_query=True)
             try:
                 list(query_iterable)
+                print("we here mate")
```

Comment:
Found you!

Reply:
I have melted laughing at this, thanks for catching it! 🤣